frontend/src/views/Home.vue

```diff
@@ -1,43 +1,85 @@
 <template>
   <div class="home flex justify-center">
-    <div class="max-w-2xl flex flex-col xl:justify-center xl:items-center">
+    <div class="w-[40rem] flex flex-col xl:justify-center xl:items-center">
       <div class="text-center text-orange-dark p-2 text-lg h-12">
-        <div v-if="criticalErrorMessage">
-          {{ criticalErrorMessage }}
+        <div v-if="errorMessage">
+          {{ errorMessage }}
         </div>
       </div>
-      <RequestDialog
-        v-if="ethereumProvider.provider"
-        :key="requestDialogReloadKey"
-        @reload="resetRequestDialog"
-      />
+
+      <div class="h-14">
+        <div v-if="signer" class="flex flex-row gap-4 justify-center items-center">
+          <div class="h-7 w-7 rounded-50 border-4 border-solid border-teal-light bg-green"></div>
+          <span class="text-lg">You are currently connected via Metamask</span>
+        </div>
+      </div>
+
+      <Card class="bg-teal mb-11 w-full min-h-[50rem]">
+        <Tabs
+          left-label="Transfer"
+          right-label="Activity"
+          :left-content-component="RequestDialog"
+          :right-content-component="null!"
+        />
+      </Card>
+
+      <div id="action-button-portal">
+        <FormKit
+          v-if="!signer"
+          input-class="w-112 bg-orange flex flex-row justify-center"
+          type="button"
+          @click="runRequestSigner"
+        >
+          <div v-if="requestSignerActive" class="h-8 w-8">
+            <spinner></spinner>
```

Comment:
Same comment for `FormKit`, but here I think because we import it as `Spinner` with capital S! I also suggest to use a self-closing tag like this `<Spinner />`

Reply:
Did I said we need better linting? 😬